{"uuid": "93ff12a2-809d-43b0-b56a-afa30d9e164c", "children": ["e67a342f-8386-46ef-8d54-2397833083ad", "ec1a8f4c-9b90-4400-b3c1-61bee78e2d57"], "befores": [{"name": "init_web_driver", "status": "passed", "start": 1689704153287, "stop": 1689704156094}], "afters": [{"name": "init_web_driver::0", "status": "broken", "statusDetails": {"message": "applitools.common.errors.DiffsFoundError: Test 'Check visually the users chart' of 'Grafana - users chart' detected differences! See details at: https://eyes.applitools.com/app/batches/00000251712596626737/00000251712596626255?accountId=uC7pT3e6R0qz0R2dpXaRFA~~\n", "trace": "  File \"C:\\Python311\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 917, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\BarKadosh-Cello\\PycharmProjects\\test_automation_final_project\\test_cases\\conftest.py\", line 66, in init_web_driver\n    eyes.close()  # Applitools\n    ^^^^^^^^^^^^\n  File \"C:\\Python311\\Lib\\site-packages\\applitools\\selenium\\eyes.py\", line 197, in close\n    return self._close(raise_ex, True)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python311\\Lib\\site-packages\\applitools\\selenium\\eyes.py\", line 533, in _close\n    results = self._commands.eyes_get_results(self._eyes_ref, raise_ex)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python311\\Lib\\site-packages\\applitools\\selenium\\command_executor.py\", line 254, in eyes_get_results\n    return self._checked_command(context, \"Eyes.getResults\", payload)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python311\\Lib\\site-packages\\applitools\\selenium\\command_executor.py\", line 267, in _checked_command\n    _check_error(response_payload)\n  File \"C:\\Python311\\Lib\\site-packages\\applitools\\selenium\\command_executor.py\", line 277, in _check_error\n    raise usdk_error\n"}, "start": 1689704185800, "stop": 1689704191655}], "start": 1689704153287, "stop": 1689704191655}